magpie/__init__.py

```diff
@@ -36,18 +36,19 @@ def __init__(self, keras_model=None, word2vec_model=None, scaler=None,
             self.scaler = scaler
 
     def train(self, train_dir, vocabulary, test_dir=None, callbacks=None,
-              nn_model=NN_ARCHITECTURE, batch_size=BATCH_SIZE,
+              nn_model=NN_ARCHITECTURE, batch_size=BATCH_SIZE, test_split=0.0,
```

Comment:
Well, usually if you wanna do things properly you would split data into independent `train` and `test` sets, so that you can validate the model on a different data that you trained on. But sometimes you might either not give a damn or just train the model on _all_ data for example before deploying into production.